docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
```

Comment:
@fvillain Could we put the docker-compose as a different PR? Maybe as a contrib/docker-compose.yml would make sense since the project mostly focuses on OpenShift.